package.json

```
@@ -81,7 +82,10 @@
     "@shardus/archiver-discovery": "^1.1.0",
     "@shardus/crypto-utils": "4.1.4",
     "@shardus/types": "1.2.21-1",
+    "@web3-storage/did-mailto": "^2.1.0",
```

let's use fixed versions
```suggestion
    "@web3-storage/did-mailto": "2.1.0",
```

---

Yes, I intend to harden it after initial tests. But yeah its not a very active repo so hardening to this version should be fine.